### PR TITLE
Capstone Project Final Code Review

### PR DESCRIPTION
**Pull request by adelphinsucasa**
This PR fullfilled:

[ x ] Add Linter
[ x ] Add HTML - CSS - JS
[ x ] JavaScript to make websites dynamic and build basic single page apps.
[ x ] Use ES6 syntax and ES6 modules.
[ x ] Use callbacks and promises.
[ x ] Use webpack.
[ x ] Follow Gitflow.
[ x ] Solve simple git conflicts.
[ x ] Send and receive data from an API.
[ x ] Use API documentation.
[ x ] Use JSON.
[ x ] Use JavaScript code asynchronous.
[ x ] Testing countComments function
[ x ] Testing characters is functional
[ x ] Add Professional Readme with Walktrough Video

### CODE REVIEW

**Comment by wayungi** [comment]
Hey, I see you are still working on this project even after submission. Please add a LICENCE file too

**Comment by adelphinsucasa** [comment]
Hi. Yes I could see a little change but is done. The MIT license file is there

**Comment by wayungi** [comment]
Pleas double check and be sure, the link in broken on my end

**Comment by adelphinsucasa** [comment]
You are right, I just changed and now is well

**Comment by wayungi** [comment]
Yes, I see the licence file is there but the link is broken. 
lic
Please use this [resource](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to add the file correctly or call me on zoom

That link does not reference the licence file in your README

**Comment by adelphinsucasa** [comment]
Yes sure. I fix the link and now is working well. thank you very much

**Comment by adelphinsucasa** [comment]
I m checking your documentation

**Comment by wayungi** [comment]
Its all good now, the licence is present now. 😄

**Comment by adelphinsucasa** [comment]
Ok now is done. Thank you, I could learn something new with your documentation :)

**Comment by wayungi** [comment]
You have deleted a lot of sections from your README. please add the steps to run your project ✍🏾 
[template](https://github.com/microverseinc/readme-template)

**Review by wayungi — COMMENTED** [review]
Hi @adelphinsucasa & @haftamudesta , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

image

## Invalid Code Review Request ❌ 

- [ ] You have submitted a project with an incomplete read-me. Please use the microverse and edit all sections. You can only delete the **FAQs** section.  Please add the steps on how to deploy & run your application [template](https://github.com/microverseinc/readme-template)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by adelphinsucasa** [comment]
It is complete in the develop branch. I just opened and all the sections are there

**Comment by wayungi** [comment]
Please compare the screen shots below to see what your file is missing

**TEMPLATE**
missing

**YOUR README**
LESS

**Comment by adelphinsucasa** [comment]
Yes. I AM fixing all the missing points. I Willrequest again soon. Thank you

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+});
+
+sothoryos.addEventListener('click', () => {
+  renderData(3);
+});
+
+ulthos.addEventListener('click', () => {
+  renderData(4);
+});
+// westeros.click();
```
Comment (rustam-tolipov) [inline src/index.js]: **You both did great job! I like the implementation!**

- [ ] However, in this line there is a commented code that need to be removed. Kindly remove it!

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello Team,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_
_Gread documentation on readme_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi team 👋🏽 , 

To highlight 👍🏽 
✔️ App runs smoothly
✔️ No commented code
✔️ README is professional

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**